docs/src/gettingstarted/introduction/own-code/cli-requirements.md

```diff
@@ -10,10 +10,12 @@ Now that you have created your free trial account, you are able to push your app
 
 Before you install it, there are some requirements that must be met first.
 
-## Git
+You need to have:
 
-[Git](/development/tools.md#git) is the open source version control system used by Platform.sh. Any change you make to your Platform.sh project will need to be committed via Git. You can see all the Git commit messages of an environment in the `Environment Activity` feed of the [management console](/administration/web/_index.md) for each project you create.
+* [PHP 5.5.9+](https://www.php.net/manual/en/install.php) with the following extensions installed: `curl`, `json`, `mbstring`, `pcre`, and `phar`.
+* [Git](https://docs.github.com/en/get-started/quickstart/set-up-git). It's is the open source version control system used to manage any changes you make to your project. You can see all the Git commit messages of an environment in the `Environment Activity` feed of the [management console](/administration/web/_index.md) for each project you create.
+* A Bash-like shell.
 
-Before getting started, make sure you [have Git installed](https://git-scm.com/) on your computer.
+On Windows, to install PHP and Git you can use [chocolatey](https://docs.chocolatey.org/en-us/). The best way to get Bash is through [Windows Subsystem for Linux](https://msdn.microsoft.com/en-gb/commandline/wsl/about).
 
-{{< guide-buttons next="I have Git ready" >}}
+{{< guide-buttons next="I have met all the requirements" >}}
```

Comment:
:pick: This is a lot less engaging than the original. Maybe something like `I have everything ready` would work?